wizard_builder/tests/test_inheritance.py

```diff
@@ -35,3 +37,31 @@ def test_conditional_instance(self):
         self.assertTrue(QuestionPage.objects.first().conditional)
         self.assertIsInstance(condition.question, MultipleChoice)
         self.assertIsInstance(condition.page, QuestionPage)
+
+
+class DumpdataHackTest(TestCase):
+
+    def test_dumpdata_hack(self):
+        QuestionPage.objects.using('test_app').create()
+
+        subprocess.check_call('''
+            python wizard_builder/tests/test_app/manage.py \
+                dumpdata \
+                    wizard_builder \
+                    -o wizard_builder/tests/test_app/test-dump.json \
+                    --natural-foreign \
+                    --indent 2
+        ''', shell=True)
+
+        subprocess.check_call('''
+            python wizard_builder/tests/test_app/manage.py \
+                loaddata \
+                    wizard_builder/tests/test_app/test-dump.json
+        ''', shell=True)
+
+        with open('wizard_builder/tests/test_app/test-dump.json', 'r') as dump_file:
+            dump_file_contents = dump_file.read()
+        self.assertIn('wizard_builder.questionpage', dump_file_contents)
+        self.assertIn('wizard_builder.pagebase', dump_file_contents)
+        self.assertEqual(PageBase.objects.using('test_app').count(), 1)
```

Comment:
Added it back, and I'm getting weird cache issues that I don't feel compelled to resolve